package.json

```diff
@@ -26,9 +26,10 @@
     "ember-cli-inject-live-reload": "^1.3.0",
     "ember-cli-qunit": "0.3.13",
     "ember-cli-uglify": "^1.0.1",
+    "ember-disable-prototype-extensions": "^1.0.0",
     "ember-disable-proxy-controllers": "^1.0.0",
     "ember-export-application-global": "^1.0.2",
-    "ember-disable-prototype-extensions": "^1.0.0",
+    "ember-legacy-views": "0.2.0",
```

Comment:
Well no, because when you include the component it won't be part of the project. Also you need to keep `disable-prototype-extension` so that people who have projects with Ember prototype extension disabled can still use this addon (it helps us find out what needs to be changed here)